device/src/esp32_run.rs

```diff
@@ -130,6 +131,12 @@ where
                                 message: format!("Failed to decode on downstream port: {e}"),
                                 device: frost_signer.device_id(),
                             });
+                            sends_upstream.push(DeviceSendMessage::Core(
+                                DeviceToCoordindatorMessage {
+                                    from: frost_signer.device_id(),
+                                    body: DeviceToCoordinatorBody::DownstreamDisconnect,
```

Comment:
Just noting that this message should be in `frostsnap_comms` not in core. What devices are "connected" is not a concept in `core`. I'm working on this issue now.